build.gradle

```diff
@@ -9,10 +9,10 @@ repositories {
     mavenCentral()
 }
 
-sourceCompatibility = 11
+sourceCompatibility = 1.8
```

Comment:
This matches httpcomponents, where we aren't able to take advantage of jep280 string concatenation improvements